flink-connector-examples/doc/flink-wordcount/README.md

```diff
@@ -67,26 +67,32 @@ taskmanager.numberOfTaskSlots: 4
 
 By default, Flink job manager rest API runs on port 8081 to receive job submission, and it is configurable via `rest.port`.
 
-Point your browser to `http://<your_flink_host>:8081` to make sure Flink is running; then click "Running Jobs"
+Point your browser to `http://<your_flink_host>:8081` to make sure Flink is running, then click "Running Jobs".
 
 ### Start WordCountWriter
-```
+
+```bash
 $ cd flink-examples/build/install/pravega-flink-examples
-$ flink run -m localhost:8081 -c io.pravega.example.flink.wordcount.WordCountWriter lib/pravega-flink-examples-0.2.0-SNAPSHOT-all.jar --host localhost --port 9999 --controller tcp://localhost:9090
+$ flink run -m localhost:8081 -c io.pravega.example.flink.wordcount.WordCountWriter lib/pravega-flink-examples-0.10.0-SNAPSHOT-all.jar --host localhost --port 9999 --controller tcp://localhost:9090
```

Comment:
I think we can remove the `-m localhost:8081` for now because it is by default done and also works when Flink cluster rest port gets deliberately changed from 8081.
For the version `0.10.0-SNAPSHOT`, I think we need to put this as `lib/pravega-flink-examples-<samples-version>-all.jar`